src/app/(proper_react)/(redesign)/(public)/how-it-works/HowItWorksView.tsx

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { ExtendedReactLocalization } from "../../../../functions/l10n";
+
+import { useL10n } from "../../../../hooks/l10n";
+
+import { Header } from "./components/Header";
+import { SectionOne } from "./components/SectionOne";
+import { SectionTwo } from "./components/SectionTwo";
+import { Footer } from "./components/Footer";
+
+export type Props = {
+  l10n: ExtendedReactLocalization;
+};
+
+export const View = () => {
+  const l10n = useL10n();
+  return (
+    <main>
+      <Header l10n={l10n} />
+      <SectionOne l10n={l10n} />
+      {/* SECTION: WE REMOVE YOUR INFO FROM DATA BROKERS */}
+      <SectionTwo l10n={l10n} />
+      {/* SECTION: WE PROVIDE STEPS TO HELP RESOLVE DATA BREACHES */}
```

Comment:
Not sure  if the comments match the components. It almost looks like the second "DATA BREACHES" comment applies to the `<Footer>` component.

Also, not sure if renaming from `<Section1>` and `<Section2>` to something more descriptive like `<DataBrokers>` and `<DataBreaches>` (which might lessen the need for comments).

Reply:
Updated here:
https://github.com/mozilla/blurts-server/pull/4632/commits/2d7459bf729335259efa338e761d1b012e695eac